reconcile/utils/jira_client.py

```diff
@@ -34,7 +34,7 @@ def get_issues(self, fields: Optional[Mapping] = None) -> GottenIssue:
         while True:
             index = block_num * block_size
             issues = self.jira.search_issues(jql, index, block_size, **kwargs)
-            all_issues.extend(issues)
+            all_issues.extend(issues)  # type: ignore[arg-type]
```

Comment:
Can we add a `TODO` here to keep this marked as under investigation / not working properly?